subsys/caf/modules/Kconfig.click_detector

```diff
@@ -17,6 +17,14 @@ config CAF_CLICK_DETECTOR_DEF_PATH
 	help
 	  Location of configuration file for click detector module.
 
+config CAF_CLICK_DETECTOR_LONG_CLICK_MSEC
+	int "Minimum button hold time to decide that the click is long"
+	default 3000
```

Comment:
default 5000

Let's keep the value as it was to ensure nothing breaks

Reply:
Good thinking! I wanted to make it 3000 as 5000 feels too long in my project, but then again, there are probably other applications where 5000 is good :) It's fixed now_